pkg/exchange/binance/exchange.go

```diff
@@ -510,7 +581,9 @@ func (e *Exchange) QueryClosedOrders(ctx context.Context, symbol string, since,
 
 		binanceOrders, err := req.Do(ctx)
 		if err != nil {
-			return orders, err
+			if !strings.Contains(err.Error(), "-1121") {
```

Comment:
what's the -1121 error code? why not just return the error?

Reply:
-1121 means symbol not found.

Reply:
fixed, thanks a lot